cmd/ak/cmd/events/dispatch.go

```diff
@@ -90,7 +96,7 @@ func init() {
 	kittehs.Must0(dispatchCmd.MarkFlagFilename("from_file"))
 
 	dispatchCmd.Flags().StringVarP(&integration, "integration", "i", "", "integration name or ID")
-	dispatchCmd.Flags().StringVarP(&connectionToken, "connection-token", "t", "", "connection token")
+	dispatchCmd.Flags().StringVarP(&connection, "connection", "C", "", "connection name or ID")
```

Comment:
From a UX perspective, we shouldn't require both an integration and a connection. Resolving the connection returns to us a connection object, which has an integration ID.

Reply:
it's also right api perspective. eliminated integration id.